djangooidc/oidc.py

```
@@ -194,39 +249,58 @@ def create_client(self, userid="", **kwargs):
         elif _key_set == set(["client_info", "srv_discovery_url"]):
             # Ship the webfinger part
             # Gather OP information
-            _ = client.provider_config(kwargs["srv_discovery_url"])
+            client.provider_config(kwargs["srv_discovery_url"])
             # register the client
-            _ = client.register(client.provider_info["registration_endpoint"],
-                                **kwargs["client_info"])
+            client.register(
+                client.provider_info["registration_endpoint"],
+                **kwargs["client_info"]
+            )
         elif _key_set == set(["provider_info", "client_info"]):
             client.handle_provider_config(
                 ProviderConfigurationResponse(**kwargs["provider_info"]),
                 kwargs["provider_info"]["issuer"])
-            _ = client.register(client.provider_info["registration_endpoint"],
-                                **kwargs["client_info"])
+            client.register(client.provider_info["registration_endpoint"],
+                            **kwargs["client_info"])
         elif _key_set == set(["provider_info", "client_registration"]):
             client.handle_provider_config(
                 ProviderConfigurationResponse(**kwargs["provider_info"]),
                 kwargs["provider_info"]["issuer"])
             client.store_registration_info(RegistrationResponse(
                 **kwargs["client_registration"]))
         elif _key_set == set(["srv_discovery_url", "client_registration"]):
-            _ = client.provider_config(kwargs["srv_discovery_url"])
-            client.store_registration_info(RegistrationResponse(
-                **kwargs["client_registration"]))
+            try:
+                client.provider_config(kwargs["srv_discovery_url"])
+                client.store_registration_info(
+                    RegistrationResponse(**kwargs["client_registration"])
+                )
+            except Exception as e:
```

This except block eats the exception, which is a change in behavior. Is this accounted for elsewhere in the code?